cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/schedule/constraint/ConstraintContext.java

```diff
@@ -47,6 +47,10 @@ public Map<ProgramRunId, RunRecordDetail> getActiveRuns(ProgramId programId) {
     return store.getActiveRuns(programId);
```

Comment:
Is anyone still using this one?

Reply:
Could not find any usages for this method, removed it.